### PR TITLE
Fixed - Database is hanging when saving API after adding multiple policies

### PR DESCRIPTION
**Pull request by shnrndk**
Fix #wso2/api-manager#699, #wso2/api-manager#618

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java [hunk 1]
```diff
@@ -18417,8 +18435,14 @@ private String addOperationPolicyContent(Connection connection, OperationPolicyD
             throws SQLException {
 
         OperationPolicySpecification policySpecification = policyData.getSpecification();
+        String toBeClonedPolicyId = policyData.getClonedCommonPolicyId();
         String dbQuery = SQLConstants.OperationPolicyConstants.ADD_OPERATION_POLICY;
-        String policyUUID = UUID.randomUUID().toString();
+        String policyUUID;
+        if (toBeClonedPolicyId == null) {
+            policyUUID = UUID.randomUUID().toString();
+        } else {
```
Comment (1akshitha) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java:18443]: You can remove this else block by directly assigning the ```policyUUID = policyData.getClonedCommonPolicyId();```

Comment (shnrndk) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java]: Fixed in #[89c143a](https://github.com/wso2/carbon-apimgt/pull/11495/commits/89c143aa9c820c3542909928a10a8189ffee0d02)

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java [hunk 2]
```diff
@@ -18417,8 +18435,12 @@ private String addOperationPolicyContent(Connection connection, OperationPolicyD
             throws SQLException {
 
         OperationPolicySpecification policySpecification = policyData.getSpecification();
+        String toBeClonedPolicyId = policyData.getClonedCommonPolicyId();
```
Comment (1akshitha) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java:18438]: Do we need this new variable? can't we directly assign policyUUID?

**Review by lakshithagunasekara — APPROVED** [review]

**Comment by HeshanSudarshana** [comment]
Add spaces between the params to maintain code readability in line 14505

**Comment by SavinduDimal** [comment]
- Remove `;` in line 18439
- Check sentence case in line 5794, 5852, 18815

**Comment by shnrndk** [comment]
@RusJaI @HeshanSudarshana 
cloneOperationPolicy() method is removed in the master branch atm.

@SavinduDimal 
New Pr For the Changes: https://github.com/wso2/carbon-apimgt/pull/12204

**Pull request merged**